DEPENDENCIES.rst

```diff
@@ -69,10 +69,10 @@ manager.
 Installing Dependencies (Linux and Unix)
 ----------------------------------------
 
-This guide assumes that the user has root access (to issue sudo commands) and
+This guide assumes that the user has root access (to issue `sudo` commands) and
 access to a package manager or has some other suitable method of automatically
 installing established libraries. This process was tested using a fresh install
-of Ubuntu 12.10 using apt-get as the package manager (scroll down further for
+of Ubuntu versions 12.10, 14.04 and 16.04, using `apt-get` as the package manager (scroll down further for
```

Comment:
I added some versions of Ubuntu here - we may want to remove the 12.10 reference